### PR TITLE
added my website

### PR DESCRIPTION
**Pull request by vyxo**

### CODE REVIEW

**Review by MaskDuck — APPROVED** [review]
nice website there!

File: domains/vyxo.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "description": "Design. Develop. Deliver",
+  "repo": "https://github.com/vyxo/vyxo.github.io",
+  "owner": {
+    "username": "vyxo"
```
Comment (MaskDuck) [inline domains/vyxo.json:5]: you need an email.

if you dont have email or don't want to share, please leave it `""` and share any of your other contact such as twitter, discord etc

**Review by MaskDuck — APPROVED** [review]

**Review by MaskDuck — CHANGES_REQUESTED** [review]
you need an email.

if you dont have email or don't want to share, please leave it `""` and share any of your other contact such as twitter, discord etc

**Comment by vyxo** [comment]
Ok, I will add my discord after configuring some changes!

**Pull request closed without merging**